skimage/feature/blob.py

```diff
@@ -415,10 +415,9 @@ def blob_log(image, min_sigma=1, max_sigma=50, num_sigma=10, threshold=.2,
     max_sigma = np.asarray(max_sigma, dtype=float)
 
     if log_scale:
-        start, stop = np.log10(min_sigma)[:, None], np.log10(max_sigma)[:, None]
-        space = np.concatenate(
-            [start, stop, np.full_like(start, num_sigma)], axis=1)
-        sigma_list = np.stack([np.logspace(*s) for s in space], axis=1)
+        start = np.log10(min_sigma)
+        stop = np.log10(max_sigma)
+        sigma_list = np.logspace(start, stop, num_sigma)
```

Comment:
my mind is blown!

Reply:
right. numpy does so much magic. I'm obsessed with this single copy paradigm, I had to try it here.

Reply:
Unfortunately, it isn't realy implemented in numpy 1.14
```
        delta = stop - start
        if num > 1:
            step = delta / div
>           if step == 0:
E           ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

In recent versions of numpy, they are now using `any`
https://github.com/numpy/numpy/blob/master/numpy/core/function_base.py#L150

I guess I'll just leave a comment there that it can be optimized when numpy gets bumped

Numpy 1.14, 1.15 still uses the logic that doesn't work for our usecase
https://github.com/numpy/numpy/blob/maintenance/1.14.x/numpy/core/function_base.py#L124
https://github.com/numpy/numpy/blob/maintenance/1.15.x/numpy/core/function_base.py#L124

Its only been fixed to numpy 1.16
https://github.com/numpy/numpy/blob/maintenance/1.16.x/numpy/core/function_base.py#L150

Reply:
Damn. It sucks not being on the bleeding edge. Thanks for the investigation!